nss-day-cohort-44/rare-rest-api-malicous-vortices
# Dw tags on post

# ISSUE TICKET 
You can add and delete tags to post. 


## Type of change

# Testing Instructions
`git fetch origin `
`git checkout  dw-tags-on-post` 
1) on postman, send a post request to http://localhost:8000/posts/1/addtag with the body of 
 {
        "tag_id": 1
    }
2) check your postTags database to see if it posted
3) send the request again and you should get a message saying that the tag is already on the post
4)  on postman, send a post request to http://localhost:8000/posts/1/addtag with the body of 
 {
        "tag_id": 2
    }
5) check your postTags database to see if it posted
6) on postman, send a get request to "http://localhost:8000/posts/1" and you should see both of the associated tags
7)  on postman, send a delete request to http://localhost:8000/posts/1/addtag with the body of 
 {
        "tag_id": 1
    }
8) check your postTags database to see if it deleted
9) on postman, send a get request to "http://localhost:8000/posts/1" and you should only see one tag


User Instructions: 

# Checklist:
